pyproject.toml

```diff
@@ -1,7 +1,92 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=67.0.0"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "pipenv"
+authors = [
+  {name = "Pipenv maintainer team", email = "distutils-sig@python.org"},
```

Comment:
The email is wrong. Please correct the email address. @matteius should our email adresses be here?

Reply:
Alright. I've been working on this PR for two weeks now and rebased multiple times. Someone else is welcome to take this on, but I've surpassed the amount of work I'm willing to put in at this time.

Reply:
We will update the email address outside this PR -- thanks for your effort on this @kalebmckale